server/src/models/Service.ts

```diff
@@ -0,0 +1,19 @@
+import { Schema, model } from 'mongoose';
+import { IService } from '../interfaces/service.interface';
+
+const ServiceSchema: Schema = new Schema(
+  {
+    staffId: { type: String, required: true },
+    name: { type: String, required: true },
+    duration: { type: Number, required: true },
+    price: { type: Number, required: true },
+  },
+  {
+    timestamps: true,
+    versionKey: false,
+  }
+);
```

Comment:
I don't understand what is this, and what is it for?

Reply:
I thought I need a schema for create the model for every collection on mongo, like here. Am I wrong?

const ClientModel = model<IClient>('Client', ClientSchema);